sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -3602,6 +3914,52 @@ public partial class PrometheusForwarderDataSource : System.ClientModel.Primitiv
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PrometheusForwarderDataSource>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.PrometheusForwarderDataSource>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public partial class Receiver : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Receiver>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>
+    {
+        public Receiver(Azure.ResourceManager.Monitor.Models.ReceiverType receiverType, string name) { }
+        public string Name { get { throw null; } set { } }
+        public string OtlpEndpoint { get { throw null; } set { } }
+        public Azure.ResourceManager.Monitor.Models.ReceiverType ReceiverType { get { throw null; } set { } }
+        public Azure.ResourceManager.Monitor.Models.SyslogReceiver Syslog { get { throw null; } set { } }
+        public Azure.ResourceManager.Monitor.Models.UdpReceiver Udp { get { throw null; } set { } }
+        Azure.ResourceManager.Monitor.Models.Receiver System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Receiver>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Monitor.Models.Receiver>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.Monitor.Models.Receiver System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Monitor.Models.Receiver>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ReceiverType : System.IEquatable<Azure.ResourceManager.Monitor.Models.ReceiverType>
```

Comment:
this name is quite generic, could we add some prefix to specify it?

Reply:
Renamed to `PipelineGroupReceiverType`